corehq/motech/openmrs/atom_feed.py

```diff
@@ -220,6 +220,10 @@ def has_new_entries_since(last_polled_at, element, xpath='./atom:updated'):
     except (RequestException, HTTPError):
         # Don't update repeater if OpenMRS is offline
         return
+    except ValueError as err:
```

Comment:
nit: this is a big scope of code in the `try` block - if possible, it'd be nice to target more narrowly where `ValueError` might come from, as it's a common exception.

Reply:
3cd4e8799dbe0e5b106d4076bfab7b5590da3024